### PR TITLE
Hannah Madison - HW01

### PR DESCRIPTION
**Pull request by hannahem**
Overall, I enjoyed completing this week’s assignment. That is largely due to that fact that I attended Thursday’s lab and received lots of help and tips from the TA and my classmates in attendance. The actual coding element of the assignment was straightforward and fun – I loved experimenting with code to produce various formatting within my bio and problem-solving to come up with a cool result. When I got stuck, my classmates helped a lot by offering paths to helpful tutorials (this one was especially helpful: https://www.rstudio.com/wpcontent/uploads/2015/03/rmarkdown-reference.pdf) The most difficult part of the assignment, for me, was figuring out the Github / RStudio workflow. However, again, my fellow classmates and the TA were very helpful in teaching me what to do. Now I am at home finishing the assignment and am able to complete the workflow on my own because of what I learned in the lab! One note on a strange occurrence – I had difficulty creating a sub-list; the formatting directions on all the R Markdown tutorials did not work to create a sub-list for me. Sub-lists should appear when two spaces are added before the star or dash, however, I had to add 4 spaces to get a sub-list. I discovered this fix through trial and error.

### CODE REVIEW

**Comment by WHAN3LSE** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |        x   |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |        x   |              |            |

This assignment is a nice piece of work which covered all the rubrics in the scoring criteria. I liked how the author presented her code in a systematic way which made everything easy to access. The information provided highlights all essential features. Overall, well done!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+- Eating at *great* restuarants
+- Spending time with fam & friends
+- Thrifting
+- Exploring Chicago
+- Taking long walks with my GoldenDoodle
+
+
+#### I am excited to gain some **solid skills** in this course! 
+
+#### Feel free to reach out on LinkedIn: (https://www.linkedin.com/in/hannahemadison/)
```
Comment (deblnia) [inline README.md:27]: Use []() syntax for in-line links. 
`[LinkedIn](https://www.linkedin.com/in/hannahemadison/)` becomes [LinkedIn](https://www.linkedin.com/in/hannahemadison/).

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      X        |            |
| **Coding strategy**         |           |     X         |            |
| **Presentation**            |     X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Successfully experiments with 4 or more elements of markdown syntax. 
* Good pull request reflection and varied commit history. 
* In re: four spaces for sub-lists: markdown can be weird sometimes! Glad you got it to work.

**Review by JinghanZhu — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    ☑️       |              |            |
| **Coding strategy**         |   ☑️        |              |            |
| **Presentation**            |    ☑️       |              |            |
| **Achievement, creativity** |     ☑️      |              |            |
| **Ease of access**          |    ☑️       |              |            |

Remarks:

* you did excellent work and met all of the requirements for the assignment, including headings, italicized text, links, and images.
* very impressed with your outfit in the Versailles. Effortless chic.
* youh have very interesting profiles. Lots of great life experience. I love film production and exploring the city as well!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+
+### Hi All! Here's a little about me: 
+* Grew up in the Seattle Area
+* Lived in the Los Angeles Area prior to MAPSS
+* I've worn many hats since undergrad:
+    * Au pairing in France
+    * Film Production
+    * Interior Architecture and Design
+    * Public Health Research
+
```
Comment (pcuppernull) [inline README.md:14]: Really cool!

File: README.md (hunk 1 above)
Comment (pcuppernull) [inline README.md:27]: Same comment as Deb, try to embed this next time

**Review by pcuppernull — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     x         |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Nice job! Really my only point of feedback is embedding the link next time. Otherwise, the photo and formatting of the other things look great.
* I didn't realize you could do sublists on your regular list. I'll definitely keep that in mind moving forward.
* In regards to your comments on getting help from the TAs and classmates, I've also asked a couple older students in my program who have taken the class before for help with a couple coding issues. Maybe you know people outside the class who could be a resource as well!

**Pull request closed without merging**